client/src/components/sources/sources.js

```diff
@@ -148,13 +152,21 @@ class Sources extends React.Component {
     });
   }
 
-  addSource() {
+  showAddSourceWizard() {
+    this.setState({ addSourceWizardShown: true });
+
+    /*
     Store.dispatch({
       type: toastNotificationTypes.TOAST_ADD,
       alertType: 'error',
       header: 'NYI',
       message: 'Adding sources is not yet implemented'
     });
+    */
+  }
+
+  hideAddSourceWizard() {
+    this.setState({ addSourceWizardShown: false });
```

Comment:
We should probably ask for confirmation so an inadvertent click outside the wizard doesn't dismiss and force the user to start over again.